src/app/lib/analyticsUtils/index.js

```diff
@@ -295,4 +301,36 @@ export const getThingAttributes = (attribute, articleData) => {
   return null;
 };
 
+export const getCampaignType = () => {
+  if (!onClient()) return null;
+
+  // Gets the query string parameters from the current url parsing them as an object
+  const { query, hash } = new Url(window.location.href, true);
+
+  // Check for the presence of the `?at_medium` QS
+  const isMediumCampaign = Object.prototype.hasOwnProperty.call(
```

Comment:
I used to think the `url-parse` utility affords us the opportunity to do something like 
```
const isMediumCampaign = new Url(window.location.href, true).query['at_medium'];
```

Reply:
It does, although this current method allows us to parse the URL once and check for multiple values e.g. check for `#at_medium` or `#xtor` without constructing the url parser twice.